codesearch/github/github.go

```diff
@@ -295,24 +295,24 @@ func ComputeIncrementalUpdate(gc GitClient, firstSha, lastSha string) (*inpb.Inc
 	var currentCommit *inpb.Commit
 	sha := firstSha
 
+	regexpSha := regexp.MustCompile("^[0-9a-f]{5,40}$")
```

Comment:
Could be moved to globals to avoid recompilation

Reply:
👍